cerberus/prometheus/client.py

```diff
@@ -2,25 +2,26 @@
 import logging
 import prometheus_api_client
 import cerberus.invoke.command as runcommand
+
 urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
 
 # Initialize the client
 def initialize_prom_client(distribution, prometheus_url, prometheus_bearer_token):
     global prom_cli
     if distribution == "openshift" and not prometheus_url:
-        url = runcommand.invoke(r"""oc get routes -n openshift-monitoring -o=jsonpath='{.items[?(@.metadata.name=="prometheus-k8s")].spec.host}'""") # noqa
+        url = runcommand.invoke(
+            r"""oc get routes -n openshift-monitoring -o=jsonpath='{
+            .items[?(@.metadata.name=="prometheus-k8s")].spec.host}'"""
```

Comment:
keep the url without newline character

url = runcommand.invoke(r"""oc get routes -n openshift-monitoring -o=jsonpath='{.items[?(@.metadata.name=="prometheus-k8s")].spec.host}'""") # noqa